api/top-langs.js

```diff
@@ -16,7 +16,8 @@ export default async (req, res) => {
     hide,
     hide_title,
     hide_border,
-    card_width,
+    card_width, // Add card_width and card_height here
```

Comment:
@airwakz can you remove these changes? These changes are not related to the pinned card. You should add your changes to https://github.com/anuraghazra/github-readme-stats/blob/master/api/pin.js.